build.sbt

```diff
@@ -31,29 +29,28 @@ lazy val microservice = Project(appName, file("."))
   .enablePlugins(Seq(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin, SbtArtifactory) ++ plugins: _*)
   .settings(playSettings: _*)
   .settings( majorVersion := 2 )
-  .configs(IntegrationTest)
   .settings(scalaSettings: _*)
   .settings(publishingSettings: _*)
   .settings(defaultSettings(): _*)
```

Comment:
You still need to include 

.settings(playSettings ++ scoverageSettings: _*)

Reply:
Applied